addons/cookoff/functions/fnc_handleDamage.sqf

```diff
@@ -21,6 +21,10 @@ _thisHandleDamage params ["_vehicle", "", "_damage", "_source", "_ammo", "_hitIn
 // it's already dead, who cares?
 if (damage _vehicle >= 1) exitWith {};
 
+// Check for players and exit if none found and the disable for AI setting is true
+private _hasPlayerInCrew = fullCrew [_vehicle, "", false] findIf {isPlayer (_x select 0)} != -1;
+if (_vehicle getVariable [QGVAR(enable), GVAR(enable)] isEqualTo 2 && !_hasPlayerInCrew) exitWith {};
```

Comment:
`if (_vehicle getVariable [QGVAR(enable), GVAR(enable)] isEqualTo 2 && {fullCrew [_vehicle, "", false] findIf {isPlayer (_x select 0)} == -1}) exitWith {};`

Reply:
That's very hard to read. What was wrong with the lines before?

Reply:
`findIf` is always checked, hd can run 30+ times for a single impact on vehicles with lots of ERA hitpoints